src/services/arweave/WarpDataProvider.ts

```diff
@@ -79,4 +96,97 @@ export class WarpDataProvider implements SmartweaveDataProvider {
     const state = await this.getContractState(id);
     return state?.balances[wallet.toString()] ?? 0;
   }
+  /**
+   *
+   * @param srcCodeTransactionId - ArweaveTransactionId of the source code used to deploy contract
+   * @param initialState - typeof ANTContractState or ArweaveTransactionID - depending on the stateType chosen, either the json initial state of the contract to deploy, or the on-chain states TXID
+   * @param stateType - type of 'TXID' | 'TAG' | 'DATA' - TXID indicates use of a prexisting state on-chain, TAG and DATA use json states locally. DATA puts the state in the data key of the transaction, TAG puts the state in the tag.
+   * @returns
+   */
```

Comment:
let's remove these for now - we can use a linter plugin to auto-generate these so they are consistent

Reply:
removed